examples/01_scale.cpp

```diff
@@ -13,7 +13,7 @@
 // Make mdspan less verbose
 using std::experimental::mdspan;
 using std::experimental::extents;
-using std::experimental::dynamic_extent;
+using std::dynamic_extent;
```

Comment:
This change suggests that CI doesn't actually build the examples.  Should we consider fixing that?

This change might actually need to depend on the C++ version, as `std::dynamic_extent` entered the Standard in C++20 with `span`.  We'll have to revisit the examples anyway, because of the recent change to use macros to specify the namespaces.  (The macros let users control them, so that they can but don't need to go into `std`.)

Reply:
CI builds the examples; I just wasn't building them on my PC.

I was unaware of the macros change. Can you give me an example?

Reply:
`MDSPAN_IMPL_STANDARD_NAMESPACE` and `MDSPAN_IMPL_PROPOSED_NAMESPACE` are the two macros.  They can be defined by users, but they also get default definitions, e.g., [here in `include/experimental/mdspan`](https://github.com/kokkos/mdspan/blob/640ca0166589bc9be7dbbeecc62c4f5edad2915c/include/experimental/mdspan#L19).  The library assumes that `MDSPAN_IMPL_PROPOSED_NAMESPACE` is nested inside `MDSPAN_IMPL_STANDARD_NAMESPACE`.

Using these macros might require updating the version of the reference mdspan implementation that github's CI pulls in.